processing/src/main/java/org/apache/druid/query/scan/ScanQuery.java

```diff
@@ -202,12 +210,38 @@ public int getBatchSize()
     return batchSize;
   }
 
-  @JsonProperty
+  /**
+   * Offset for this query; behaves like SQL "OFFSET". Zero means no offset. Negative values are invalid.
+   */
+  @JsonProperty("offset")
+  @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+  public long getScanRowsOffset()
+  {
+    return scanRowsOffset;
+  }
+
+  /**
+   * Offset for this query; behaves like SQL "LIMIT". Will always be positive. {@link Long#MAX_VALUE} is used in
```

Comment:
Same comment as https://github.com/apache/druid/pull/10235#discussion_r469416176. :-)

Reply:
Yes, you're right, fixed.